src/modules/task/dto/update-task.dto.ts

```diff
@@ -0,0 +1,12 @@
+import 'reflect-metadata';
+import { IsNotEmpty, IsString } from 'class-validator';
+
+export class UpdateTaskDto {
+  @IsNotEmpty()
+  @IsString()
+  title: string;
+
+  @IsString()
+  @IsNotEmpty()
+  description: string;
```

Comment:
Это Update дто, а значит могут обновлять сущность частично, например передавать только новый title или только новый description. Поэтому оба поля должны быть опциональными
```suggestion
  @IsOptional()
  @IsString()
  title?: string;

  @IsString()
  @IsOptional()
  description?: string;
```